react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -218,6 +224,49 @@ class AddPeopleDialog extends Component {
                 });
             });
         }
+
+        return true;
+    }
+
+    /**
+     * Renders the error message if the add doesn't succeed.
+     *
+     * @returns {ReactElement|null}
+     * @private
+     */
+    _renderErrorMessage() {
+        if (!this.state.addToCallError) {
+            return null;
```

Comment:
I actually agree and this is more intuitive for me. I was seeing code written this way and was trying to follow the "convention", but if there's no decision on that I would prefer the other way as well.